xml/art_jeos_quickstart.xml

```diff
@@ -301,8 +301,145 @@
    <package>cloud-init-config-suse</package>, and
    <package>xfsprogs</package>.
   </para>
- </sect2>
+  <sect3 xml:id="sec-cloud-init-intro">
+  <title>cloud-init basics</title>
+  <para>
+  cloud-init applies automatically custom configuration to running instances.
+  The tool features a wide range of configuration modules, and it can execute
+  specific commands in the image during the boot phase.
```

Comment:
Image of what? 

I am not sure myself so I think that maybe this does need clarification. E.g. on my home RasPi server, I found cloud-init was present and running by default, on bare metal. From this I deduce that cloud-init is not just related to VMs or containers.